src/openforms/contrib/kadaster/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+
+def get_default_kadaster_service():
+    default_service = Service.objects.create(
+        label="Locatieserver",
+        api_type=APITypes.orc,
+        api_root="https://api.pdok.nl/bzk/locatieserver/search/",
+        oas="https://api.pdok.nl/bzk/locatieserver/search/v3_1/openapi.json",
+        auth_type=AuthTypes.no_auth,
+    )
```

Comment:
this should be a `get_or_create` based on `api_root` - if a service already exists with this API root, this will crash with `IntegrityError` due to a database unique constraint being violated.